flake.nix

```diff
@@ -57,8 +57,8 @@
         lib = nixpkgs.lib;
         iceflowPackage = ({crossTarget ? "${system}"}:  let
           pkgs = (import nixpkgs { localSystem = system; crossSystem = crossTarget; }).extend self.overlays.default;
-        in 
-          pkgs.callPackage ({enableExamples ? false, enableGRPC ? true, stdenv, pkgs}: stdenv.mkDerivation {
+        in
+          pkgs.callPackage ({enableExamples ? false, enableGRPC ? false, stdenv, pkgs}: stdenv.mkDerivation {
```

Comment:
I guess that should have already done the trick, right?

Reply:
You probably want to enable GRPC in the built docker images if you disable it by default (otherwise, the published images won't be usable with the node executor).

Reply:
_Should_ be adjusted via c75bc08f97e6856ce7b3bc157a7dd2835db942bb now.